### PR TITLE
add support for SSH with isolated site users

### LINKED ISSUE #20: SSH with site isolation
It would be awesome if the `forge ssh` command took into account the isolated system user account rather than assuming `forge`.

Is that something you’d consider adding?

### PR DESCRIPTION
**Pull request by macbookandrew**
Closes #20

### CODE REVIEW

File: app/Commands/SshCommand.php [hunk 1]
```diff
@@ -33,6 +34,14 @@ public function handle()
             ]);
         }
 
+        $name = $this->argument('site');
+
+        if (! is_null($name)) {
+            $answers = collect($this->forge->sites($this->currentServer()->id));
```
Comment (taylorotwell) [inline app/Commands/SshCommand.php:40]: answers?

Comment (macbookandrew) [reply app/Commands/SshCommand.php]: Oops…copied from elsewhere and forgot to change the name of the variable. Fixed now

**Comment by nunomaduro** [comment]
I am sorry, we are going to delay this suggestion for a couple of days.

**Comment by macbookandrew** [comment]
@nunomaduro thanks for considering this. Just curious if you have plans for any other mechanism for running `artisan` commands on a specific site—that’s my main use case for this.

**Comment by nunomaduro** [comment]
Well, you can use the "command" command. https://forge.laravel.com/docs/1.0/accounts/cli.html#running-commands.

**Comment by macbookandrew** [comment]
I hadn’t thought about that, but that doesn’t allow for any interaction with the command (e.g., `forge command --command 'php artisan vendor:publish'` just sits “waiting for command to run”).

(Yes, I realize that specific command is intended for a development and not a deployed environment, but it makes a decent test case.)

**Pull request closed without merging**